stacks/main.py

```diff
@@ -61,6 +69,7 @@ def main():
     parser_events = subparsers.add_parser('events', help='List events from a stack')
     parser_events.add_argument('name')
     parser_events.add_argument('-f', '--follow', action='store_true', help='Poll for new events until stopped')
+    parser_events.add_argument('-l', '--lines', default='10', type=int)
```

Comment:
Usually, `-n, --lines` is used instead of `-l`. `journalctl`, `tail`, `head` uses that as well.